### PR TITLE
Appveyor: Add Werror to msvc builds

### PR DESCRIPTION
**Pull request by 1480c1**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -24,7 +28,7 @@ deploy:
   - provider: GitHub
     artifact: $(APPVEYOR_PROJECT_NAME)
     auth_token:
-      secure: 'sf0pQXlPI+X6LoAR8QUJB74jjzNxcLGOXI3H0nbxJq8llvGPG/TAUd87hq5iHZXo'
+      secure: "sf0pQXlPI+X6LoAR8QUJB74jjzNxcLGOXI3H0nbxJq8llvGPG/TAUd87hq5iHZXo"
```
Comment (EwoutH) [inline appveyor.yml]: Why the change in quotation marks?

Comment (1480c1) [reply appveyor.yml]: Auto formatter, single and double quotes are same it seems

**Comment by hassount** [comment]
@1480c1 @EwoutH 
Thank you both :)
